modules/sandboxed-containers-viewing-workloads-from-web-console.adoc

```diff
@@ -3,5 +3,23 @@
 // * sandboxed_containers/deploying_sandboxed_containers.adoc
 
 [id="sandboxed-containers-viewing-workloads-from-web-console_{context}"]
+= Viewing {sandboxed-containers-first} workloads from the web console
 
-= Viewing sandboxed containers workloads from the web console
+{sandboxed-containers-first} based workloads look and feel the same as normal workloads when viewed in the web console. The only difference between the two is the `runtimeClassName`. `runtimeClassName` is what decides the runtime used for workloads. In this context, the runtime enabled by {sandboxed-containers-first}-based is `kata`. You can view the `runtimeClass` that the pods for your workloads use.
+
+.Prerequisites
+
+* You have {product-title} {product-version} installed on your cluster.
+* You have access to the cluster as a user with the `cluster-admin` role.
```

Comment:
I don't think you need `cluster-admin` to simply view resources. I think `view` should be enough here. @zanetworker @jensfr ?

Reply:
Yes, this could be done with the view role. But if cluster-admin is used everywhere else in OpenShift doc we should do the same here? 

Maybe it's common practice to give developers cluster-admin in their projects which boils down to admin "plus a few additional permissions like the ability to edit rate limits, for that project"?  (quote from the documentation: https://docs.openshift.com/container-platform/4.7/authentication/using-rbac.html#default-roles_using-rbac)

Reply:
@zanetworker @jensfr Does the user need an admin role to view the Administration perspective in the console?